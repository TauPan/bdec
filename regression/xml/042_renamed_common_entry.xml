<protocol>
    <sequence name="header">
        <reference name="length" type="number" />
        <field name="data" length="(${length} * 8)" type="text" />
    </sequence>
    <common>
        <sequence name="number" value="(${digit:} - 6 * 8)">
            <field name="digit:" length="8" type="integer" />
        </sequence>
    </common>
</protocol>
