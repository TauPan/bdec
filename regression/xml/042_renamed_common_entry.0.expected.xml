<header> <length>5</length> <data>abcde</data> </header>